.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '^cuper.*'
+  pull_request:
+    branches: [master]
+
+env:
+  BUILD_DIR: ${{ github.workspace }}/build
+
+jobs:
+  build:
+    name: Build QEMU for Linux
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install apt dependency
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends \
+            bison \
+            flex \
+            gettext \
+            help2man \
+            libncurses5-dev \
+            libtool-bin \
+            libtool-doc \
+            meson \
+            ninja-build \
+            tar \
+            texinfo
+      - name: Check out source code
+        uses: actions/checkout@v2
+      - run: git fetch --prune --unshallow --tags --force
+
+      - name: Build QEMU
+        env:
+          QEMU_TARGETS: arc-linux-user arc64-linux-user arc-softmmu arc64-softmmu
+          QEMU_FLAGS: |
+            --enable-system \
+            --enable-linux-user \
+            --disable-werror \
```

Comment:
Hmm, is it OK to just ignore all the warnings by passing `--disable-werror`?

Reply:
@evgeniy-paltsev better ask the maintainer why the can't be compiled with werror flag. Even changing this line want help in merging this PR.